policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit_base_cap.py

```diff
@@ -0,0 +1,67 @@
+from policyengine_us.model_api import *
+
+
+class me_property_tax_fairness_credit_base_cap(Variable):
+    value_type = float
+    entity = TaxUnit
+    unit = USD
+    label = "Maine property tax fairness credit base cap"
+    definition_period = YEAR
+    defined_for = StateCode.ME
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.me.tax.income.credits.fairness.property_tax
+        dependents = tax_unit("ctc_qualifying_children", period)
+        filing_status = tax_unit("filing_status", period)
+        filing_statuses = filing_status.possible_values
+        single = filing_status == filing_statuses.SINGLE
+        joint_no_child = (filing_status == filing_statuses.JOINT) & (
+            dependents == 0
+        )
+        head_of_household_one_child = (
+            filing_status == filing_statuses.HEAD_OF_HOUSEHOLD
+        ) & (dependents <= p.dependent_count_threshold)
+        joint_with_child = (filing_status == filing_statuses.JOINT) & (
+            dependents >= p.dependent_count_threshold
+        )
+        head_of_household_multiple_child = (
+            filing_status == filing_statuses.HEAD_OF_HOUSEHOLD
+        ) & (dependents > p.dependent_count_threshold)
+        # legal code does not mention widow, but it does appear in the tax form
+        widow = filing_status == filing_statuses.SURVIVING_SPOUSE
+        benefit_base = select(
```

Comment:
Also I think me_property_tax_fairness_credit_eligible is missing this limitation, could you add it?
```suggestion
        # Separate filers are ineligible so not included in this select statement.
        benefit_base = select(
```

Reply:
`me_property_tax_fairness_credit_eligible` uses the `income_limit.yaml`  parameter, where the values for separate are all zeros, it does make separate filers ineligible. 
This brings a new question, since the `income_limit.yaml` has the same structure as the `amount.yaml` (replaced by benefit_base folder), do I need to reorganize the `income_limit.yaml` as well? I am still trying to find legal references for those values.

Reply:
See if income limits are derived from other parameters before looking more for legal code

Reply:
It comes from the benefit base, benefit_base/rate(4% in 2022) = income_limit.